specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/CommonDefinitions.json

```diff
@@ -160,6 +175,11 @@
         },
         "networkingConfiguration": {
           "$ref": "#/definitions/AseV3NetworkingConfiguration"
+        },
+        "upgradeAvailable": {
```

Comment:
>upgradeAvailable [](http://example.com/codeflow?start=9&length=16)

Recommend enum in favor of booleans. They are more descriptive, flexible and future-proof while being easier for customers to discover, understand and use. Ref: [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html?q=boolean#common-issues-found-in-review).

You could use something like upgrades: { None, Pending }.